Project.toml

```
@@ -1,13 +1,14 @@
 name = "FHIRClient"
 uuid = "b44d2ca2-8176-4fa9-8684-826e17b2a2da"
 authors = ["Dilum Aluthge", "Rhode Island Quality Institute", "contributors"]
-version = "2.2.0"
+version = "2.3.0"
```

There are a couple changes on master now, so let's avoid touching the version number here, and then we can bump the version in a separate standalone PR.

```suggestion
version = "2.2.0"
```